docs/configuration/parsing-methods.rst

```diff
@@ -75,6 +77,23 @@ To use this:
         # ...,
     )
 
+``dbt_ls_file``
+----------------
+
+If you provide the output of ``dbt ls --output json`` as a file, you can use this to parse similar to  ``dbt_ls``.
+You can supply a ``dbt_ls_path`` parameter on the DbtDag / DbtTaskGroup with a path to a ``dbt_ls_output.txt`` file.
+
+To use this:
+
+.. code-block:: python
+
+    DbtDag(
```

Comment:
As mentioned in the broader comment, if we could update a DAG in `dev/dags` and add a reference to that runnable DAG code here, similar to:
https://github.com/astronomer/astronomer-cosmos/blob/main/dev/dags/basic_cosmos_dag.py#L24
it would be awesome.

Since the example DAGs are always run as part of our test suite, we'd guarantee this excerpt doesn't become outdated.

Reply:
I've updated the dag you mentioned in the broader comment. Hope that is what you had in mind?